docs/how-to/setting-cus.rst

```diff
@@ -0,0 +1,42 @@
+.. meta::
+    :description: Setting the number of CUs
+    :keywords: CU, CUs, number of CUs, compute units
+
+.. _env-variables-reference:
+
+*************************************************************
+Setting the number of compute units
+*************************************************************
+
+.. tip::
```

Comment:
Not good practice to start with a tip. 
Remove the tip box.

A better alternative is to move the tip box below the first two bullets.